atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/qos/QosCassandraClient.java

```diff
@@ -67,18 +67,21 @@ public QosCassandraClient(CassandraClient client, QosClient qosClient) {
     public Map<ByteBuffer, List<ColumnOrSuperColumn>> multiget_slice(String kvsMethodName, TableReference tableRef,
             List<ByteBuffer> keys, SlicePredicate predicate, ConsistencyLevel consistency_level)
             throws InvalidRequestException, UnavailableException, TimedOutException, TException {
+        int numberOfQueriedRows = keys.size();
```

Comment:
i would rather give the parameters to the weigher, and have it extract the size (the weigher is responsible for determining weight, not this class)